src/pymbus/codes/vif.py

```diff
@@ -64,781 +69,822 @@ class VIFCodeUnit(str, Enum):
     ),
     0b0001_0011: VIFCode(
-        code=0x13,
+        # code=0x13,
         coef=1e-3,
-        desc=VIFCodeDescription.volume,
+        kind=VIFCodeKind.volume,
         unit=VIFCodeUnit.meter_cubic,
     ),
     0b0001_0100: VIFCode(
-        code=0x14,
+        # code=0x14,
         coef=1e-2,
-        desc=VIFCodeDescription.volume,
+        kind=VIFCodeKind.volume,
         unit=VIFCodeUnit.meter_cubic,
     ),
     0b0001_0101: VIFCode(
-        code=0x15,
+        # code=0x15,
         coef=1e-1,
-        desc=VIFCodeDescription.volume,
+        kind=VIFCodeKind.volume,
         unit=VIFCodeUnit.meter_cubic,
     ),
     0b0001_0110: VIFCode(
-        code=0x16,
+        # code=0x16,
         coef=1e0,
-        desc=VIFCodeDescription.volume,
+        kind=VIFCodeKind.volume,
         unit=VIFCodeUnit.meter_cubic,
     ),
     0b0001_0111: VIFCode(
-        code=0x17,
+        # code=0x17,
         coef=1e1,
-        desc=VIFCodeDescription.volume,
+        kind=VIFCodeKind.volume,
         unit=VIFCodeUnit.meter_cubic,
     ),
     # E001_1nnn - Mass (Kilogram = kg)
     0b0001_1000: VIFCode(
-        code=0x18,
+        # code=0x18,
         coef=1e-3,
-        desc=VIFCodeDescription.mass,
+        kind=VIFCodeKind.mass,
         unit=VIFCodeUnit.kilogram,
     ),
     0b0001_1001: VIFCode(
-        code=0x19,
+        # code=0x19,
         coef=1e-2,
-        desc=VIFCodeDescription.mass,
+        kind=VIFCodeKind.mass,
         unit=VIFCodeUnit.kilogram,
     ),
     0b0001_1010: VIFCode(
-        code=0x1A,
+        # code=0x1A,
         coef=1e-1,
-        desc=VIFCodeDescription.mass,
+        kind=VIFCodeKind.mass,
         unit=VIFCodeUnit.kilogram,
     ),
     0b0001_1011: VIFCode(
-        code=0x1B,
+        # code=0x1B,
         coef=1e0,
-        desc=VIFCodeDescription.mass,
+        kind=VIFCodeKind.mass,
         unit=VIFCodeUnit.kilogram,
     ),
     0b0001_1100: VIFCode(
-        code=0x1C,
+        # code=0x1C,
         coef=1e1,
-        desc=VIFCodeDescription.mass,
+        kind=VIFCodeKind.mass,
         unit=VIFCodeUnit.kilogram,
     ),
     0b0001_1101: VIFCode(
-        code=0x1D,
+        # code=0x1D,
         coef=1e2,
-        desc=VIFCodeDescription.mass,
+        kind=VIFCodeKind.mass,
         unit=VIFCodeUnit.kilogram,
     ),
     0b0001_1110: VIFCode(
-        code=0x1E,
+        # code=0x1E,
         coef=1e3,
-        desc=VIFCodeDescription.mass,
+        kind=VIFCodeKind.mass,
         unit=VIFCodeUnit.kilogram,
     ),
     0b0001_1111: VIFCode(
-        code=0x1F,
+        # code=0x1F,
         coef=1e4,
-        desc=VIFCodeDescription.mass,
+        kind=VIFCodeKind.mass,
         unit=VIFCodeUnit.kilogram,
     ),
     # E010_00nn - On time (in seconds)
     0b0010_0000: VIFCode(
-        code=0x20,
+        # code=0x20,
         coef=1,
-        desc=VIFCodeDescription.on_time,
+        kind=VIFCodeKind.on_time,
         unit=VIFCodeUnit.second,
     ),
```

Comment:
And this one is also measurable, but not all may agree about this code to be categorised as a physical unit.